scripts/get_most_recent_hashes.py

```diff
@@ -0,0 +1,56 @@
+# Gets the 100 most recent or subsequent hashes.
+
+import os
+import argparse
+
+
+def parse_arguments():
+    """Parse command line arguments"""
+    parser = argparse.ArgumentParser(description="Get closest valid GCC hash")
+    parser.add_argument(
+        "-hash",
+        required=True,
+        type=str,
+        help="Commit hash of GCC to get artifacts for",
+    )
+    parser.add_argument(
+        "-subsequent",
+        action="store_true",
+        help="Get the subsequent closest commit. Defaults to prior.",
+    )
+    return parser.parse_args()
+
+
+def gcc_hashes(git_hash: str, subsequent: bool):
```

Comment:
I think if we aren't using anything inside of this file except for this function and not actually calling this file inside of any yaml files, might as well just move this into `download_artifacts.py`